tests/test_indices.py

```diff
@@ -29,6 +29,11 @@
 from xclim.core.options import set_options
 from xclim.core.units import convert_units_to, units
 
+try:
+    import flox
+except ImportError:
+    flox = None
```

Comment:
Would it be better if we were adding `flox` to the `tox` configuration, then `parametrizing` the tests to use `flox` if it's there? I believe we have one build that tests the `extras` recipe already.

Reply:
Yes, but my current "testing" only consist in hijacking an existing test and enabling the new option in a certain case. A `importorskip('flox')` would be much better, but I'll do that on a test specific to `resample_map`. To come in the next working days...

Reply:
Okay, that sounds fine to me! Just keep it in mind.